marlowe-chain-sync/.golden/ChainSeek/golden

```diff
@@ -0,0 +1,516 @@
+Show: MsgQueryNext (AdvanceBlocks 1)
```

Comment:
Should these really be in hidden folders? That might mystify users.

Reply:
This is where `hspec-golden` puts the golen test data files by default - though we could certainly change it.